main.py

```diff
@@ -223,6 +233,7 @@ def main():
     datadir=options.datadir,
     debug=DEBUG,
     template_path=os.path.join(os.path.dirname(__file__), "templates"),
+    password=UPLOAD_PASSWORD
```

Comment:
Missing trailing comma. Python is better than old JS in this aspect :-)

Reply:
done